tailwind.config.js

```diff
@@ -33,6 +33,9 @@ module.exports = {
         red: theme('colors.red'),
         white: theme('colors.white')
       }),
+      height: {
+        case: '113.5%'
```

Comment:
@hannakim91 this seems like a very specific number, could you please explain it?

Reply:
@hannakim91 Yeah I fixed this here [Fix image gap](https://github.com/shipshapecode/website-nuxt/pull/222/commits/9244c2a073f7413da143d83826790ebe9adfdc6c)